modules/ting_field_search/plugins/ting_field_search_ctools_export_ui.inc

```diff
@@ -158,11 +160,61 @@ function ting_field_search_profile_export_form(&$form, &$form_state) {
     '#default_value' => $profile->config['user_interaction']['alter_links'],
   );
 
+  $form['config']['user_interaction']['grouping'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Profile grouping'),
+    '#description' => t('Which group of search profiles should be shown when this profile is used.'),
```

Comment:
I do not understand what this description means regarding groups of profiles to be shown when this profile is used. I cannot tell how a profile is added to a group.

Reply:
Simply grouped by the string/name entered in the field.